src/server/saved_objects/service/lib/repository.js

```diff
@@ -606,4 +614,10 @@ export class SavedObjectsRepository {
     const savedObject = this._serializer.rawToSavedObject(raw);
     return omit(savedObject, 'namespace');
   }
+
+  assertAllowedType(type) {
```

Comment:
nit: if this isn't going to be publicly exposed, we should make it `_assertAllowedType` to remain consistent with other "private" methods in the SavedObjectsRepository.

Reply:
Not a problem, I can add the underscore to stay consistent. Personally I'm not a fan of mnemonics or prefixes like that but since there are not really any *good* ways of making true private members the underscore does serve a purpose here.

Reply:
Agreed, but since we're doing so here, I'd prefer we at least remain consistent.